Tribler/Core/Utilities/random_utils.py

```diff
@@ -0,0 +1,44 @@
+import string
+
```

Comment:
Please remove this space.

Reply:
Also, the order of the `string` and `random` imports should be reverted.